### PR TITLE
doc/SubmittingPatches: Fix 'Fixes line' hyperlink

### PR DESCRIPTION
**Pull request by varshar16**
Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: SubmittingPatches.rst [hunk 1]
```diff
@@ -133,7 +133,7 @@ Some negative examples (how *not* to title a commit message)::
      bug fix for driver X
      fix issue 99999
 
-Further to the last negative example ("fix issue 99999"), see `Fixes line`_.
+Further to the last negative example ("fix issue 99999"), see `Fixes line(s)`_.
```
Comment (tchaikov) [inline SubmittingPatches.rst]: i think Nathan intended to put "fixes line" instead. as he added an anchor right before the section of "Fixes line(s)".

File: SubmittingPatches.rst [hunk 2]
```diff
@@ -133,7 +133,7 @@ Some negative examples (how *not* to title a commit message)::
      bug fix for driver X
      fix issue 99999
 
-Further to the last negative example ("fix issue 99999"), see `Fixes line`_.
+Further to the last negative example ("fix issue 99999"), see :ref:`fixes line`.
```
Comment (varshar16) [inline SubmittingPatches.rst:136]: I have tried both 'fixes line'_ and :ref:'fixes line'. Both don't work. We
cannot use ref as anchor is defined in later part. I am reverting back to
original commit.

**Comment by smithfarm** [comment]
@varshar16 Can you also drop the anchor line (if it doesn't work as intended), so it doesn't confuse anyone?

**Comment by varshar16** [comment]
> @varshar16 Can you also drop the anchor line (if it doesn't work as intended), so it doesn't confuse anyone?

Done.

**Pull request merged**